Gemfile.lock

```diff
@@ -3,7 +3,7 @@ PATH
   specs:
     ten_years_rails (1.0.1)
       actionview (~> 5.2.3)
-      colorize (>= 0.8.1)
+      rainbow (~> 3.0.0)
```

Comment:
Gem review:

✅  Gem has appropriate license ([MIT](https://github.com/sickill/rainbow/blob/master/LICENSE))
✅ Gem appears to be widely used (used by 78k+ repos according to GitHub)
✅ Gem has no significant outstanding vulnerabilities (none, but [searching for rainbow](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=rainbow) is pretty fun in a CVE database 🤣)
✅  Code is of reasonable quality
✅  Your squad(*) commits to keeping the gem up to date on a regular basis

Reply:
Nothing found in https://github.com/rubysec/ruby-advisory-db/search?q=rainbow&unscoped_q=rainbow

Reply:
👍 Endorsed the gem review.